WORKSPACE

```diff
@@ -466,3 +466,12 @@ maven_jar(
   artifact = "com.yahoo.pulsar:pulsar-client:jar:shaded:1.18"
 )
 # end Pulsar Client
+
+# for pex rules
+git_repository(
+    name = "io_bazel_rules_pex",
+    remote = "https://github.com/streamlio/bazel_rules_pex.git",
```

Comment:
you can have a copy of that repo internally at Twitter. In twitterify script, you could change the repo to refer to the internal repo.